### ISSUE TITLE
Patch line shown on the probability maps

### ISSUE THREAD

**Issue by a3sha2** [ANTsRNet/BrainSegmentationPatchBased#4]
The path line is shown on the maps
![Screen Shot 2019-10-08 at 1 04 27 PM](https://user-images.githubusercontent.com/8614410/66417180-74e14a80-e9cd-11e9-8d22-64a23651d0c9.png)

**Comment by ntustison** [comment]
Yeah, there are various approaches to mitigate these artifacts like estimating multiple patch sets but those aren't included in the current script.
